modules/op-pac-additional-controller.adoc

```diff
@@ -0,0 +1,36 @@
+// This module is included in the following assemblies:
+// * pac/install-config-pipelines-as-code.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="pac-additional-controller_{context}"]
+= Configuring additional {pac} controllers to support additional GitHub apps
+
+[role="_abstract"]
+By default, you can configure {pac} to interact with one GitHub app. In some cases you might need to use more than one GitHub app, for example, if you need to use different GitHub accounts or different GitHub instances such asGitHub Enterprise or GitHub SaaS. If you want to use more than one GitHub app, you must configure an additional {pac} controller for every additional GitHub app.
+
+.Procedure
+
+. In the `TektonConfig` custom resource, in the `pipelinesAsCode` spec, add the `additionalPACControllers` section, as in the following example:
++
+.Example `additionalPACControllers` section
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  pipelinesAsCode:
+    additionalPACControllers:
+      controllername: pac_controller_2  # <1>
+        enable: true    # <2>
+        secretName: pac_secret_2  # <3>
+        settings: #  <4>
```

Comment:
Thanks a lot! I fixed this now, and the absence of "platforms.openshift " in the spec is also a bug in existing documentation. I have created a separate Jira for that bug as opposed to fixing it in this PR because it will need to be merged to the previous versions too.